doc-content/enterprise-only/openshift/template-deploy-start-proc.adoc

```diff
@@ -64,6 +56,21 @@ endif::DM[]
 :template_add_params: -p KIE_SERVER_HTTPS_SECRET=kieserver-app-secret
 endif::[]
 
+ifeval::["{context}"=="server-immutable-s2i"]
+:single_template_file!:
+:template_add_params: -p KIE_SERVER_HTTPS_SECRET=kieserver-app-secret
+ifdef::PAM[]
+:modifylink: environment-immutable-modify-proc
+:modifytype: fixed
+endif::PAM[]
+:template_name: <template-file-name>.yaml
+To deploy {what_deploying}, use one of the following template files:
+
+* If you want to enable JMS capabilities of the immutable {KIE_SERVER}, use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver-amq.yaml` template file.
+* Otherwise, use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-prod-immutable-kieserver.yaml` template file.
```

Comment:
There are two issues in this bulleted list. 1) They don't have a consistent tone. 2)Not sure if the user actually has a choice here since the first one is to be used only in case the user wants JMS capabilities.
Suggestion: Can we get rid of the bullet and just say:
"To deploy an immutable Process Server using an S2I build, use rhpam74-prod-immutable-kieserver-amq.yaml template file to enable JMS capabilities. Otherwise use the rhpam74-prod-immutable-kieserver.yaml template file."
WDYT?

Reply:
Made the change, but kept "if you want", because "otherwise" is grammatically hooked to it.